matter_server/client/client.py

```diff
@@ -122,7 +122,13 @@ async def commission_with_code(self, code: str) -> MatterNodeData:
 
     async def commission_on_network(self, setup_pin_code: int) -> MatterNodeData:
         """
-        Commission a device already connected to the network.
+        Do the routine for OnNetworkCommissioning, with a filter for mDNS discovery.
+
+        The filter can be an integer,
```

Comment:
There's no filter in the parameters on the client side. It's not clear what we're talking about.

Reply:
I copied the docstring from the sdk but we shouldn't be using this method at all.
We should check our iOS and Android app if they're not using it and then just remove it. I was told this is only used for some of the CSA tests and all normal commissioning is just handled by the other command, which accepts all codes.

For now, I'll just adjust the docstring in the client to deprecated.

Reply:
updated the docstring